sdcm/cluster.py

```diff
@@ -4492,6 +4493,7 @@ def node_setup(self, node, verbose=False, db_node_address=None, **kwargs):  # py
 
         if 'scylla-bench' in self.params.list_of_stress_tools:
             self.install_scylla_bench(node)
+            self.install_cassandra_harry(node)
```

Comment:
How is it related to scylla-bench?
Why would we want to  install cassandra-harry everytime we have scylla-bench?

We need to  install it only when we need to.

Reply:
fixed.  only install/update cassandra-harry when it exists in the stress_cmd